src/uu/install/src/install.rs

```diff
@@ -5,7 +5,10 @@
 //  * For the full copyright and license information, please view the LICENSE file
 //  * that was distributed with this source code.
 
-// spell-checker:ignore (ToDO) rwxr sourcepath targetpath
+// spell-checker:ignore (ToDO) rwxr sourcepath targetpath Isnt uioerror
+
+// clippy bug https://github.com/rust-lang/rust-clippy/issues/7422
+#![allow(clippy::nonstandard_macro_braces)]
```

Comment:
You can remove these lines again, the clippy bug has been fixed.

Reply:
Done!